packages/deploy-utils/src/utils/git/git.test.ts

```diff
@@ -71,10 +71,8 @@ describe("getCurrentlyDeployedCommit", () => {
 
   it("returns a valid local commit", async () => {
```

Comment:
It might be interesting to see if `getCurrentlyDeployedCommit` could be tested by executing it for real, I think the `getCurrentlyDeployedCommit` from `apps/spruce/scripts/deploy/utils/git` is tested without any mocking :O just an observation though I'm not sure if that's difficult